src/components/Signup.jsx

```diff
@@ -51,8 +47,15 @@ const Signup = () => {
     });
   };
 
-  //   if (loginStatus)
-  //     return <Redirect to={{ pathname: '/explore', state: { username } }} />;
+  if (registerStatus)
+    return (
+      <Redirect
+        to={{
+          pathname: '/explore',
+          state: { username: registrationInputs.username },
+        }}
+      />
```

Comment:
Subject to refactor. Just playing around with how to keep track of username in subsequent views in user flow. - Andy Wang